modules/carto/src/index.js

```diff
@@ -1,3 +1,4 @@
 export {getDefaultCredentials, setDefaultCredentials} from './auth.js';
 export {default as CartoSQLLayer} from './layers/carto-sql-layer';
 export {default as CartoBQTilerLayer} from './layers/carto-bqtiler-layer';
+export {default as basemaps} from './basemaps';
```

Comment:
deck.gl uses all CAPS for constant names.

Reply:
Done! Thanks for the tip!